src/doc/examples/mapboxstyle.js

```diff
@@ -11,15 +11,15 @@ export default () => {
     }),
   });
 
-  const url = `https://maps.geops.io/styles/base_bright_v1/style.json?key=${window.apiKey}`;
+  const url = `https://maps.geops.io/styles/base_bright_v2/style.json?key=${window.apiKey}`;
   const mapboxLayer = new MapboxLayer({ url });
 
   const poiLayer = new MapboxStyleLayer({
     name: 'poi layer',
     visible: true,
     mapboxLayer,
     styleLayer: {
-      id: 'poi_with_icons-lvl',
+      id: 'poi_with_icons',
```

Comment:
Why here? Why not in query-objects.js? What's the difference?

Reply:
I could change as well in query-object example.
The 'poi_with_icons-lvl' is a specific styleLayer for trafimage-level-switcher, and is by default not visible.
The normal layer is now 'poi_with_icons'